pyproject.toml

```diff
@@ -76,7 +76,7 @@ thycotic_tss = "awx_plugins.credentials.tss:tss_plugin"
 aws_secretsmanager_credential = "awx_plugins.credentials.aws_secretsmanager:aws_secretmanager_plugin"
 
 [project.entry-points."awx_plugins.inventory"]  # new entry points group name
-azure-rm = "awx_plugins.inventory.plugins:azure_rm"
+azure_rm = "awx_plugins.inventory.plugins:azure_rm"
```

Comment:
The tests must be adjusted for this, so they don't fail.

Reply:
Which tests? When I point the requirements for awx at this branch on my repo the related tests pass. I'll run them again to confirm.

Reply:
Ones failing in CI — smoke: https://github.com/ansible/awx-plugins/blob/8e8d03b/tests/importable_test.py#L76 / https://github.com/ansible/awx-plugins/actions/runs/12121157958/job/33791551145?pr=61#step:17:51